File: config/socket_config.py
from collections import namedtuple
import json

SocketConfig = namedtuple('SocketConfig', ['host', 'port'])

with open('config/server-config.json') as file:
    socket_config = json.load(file)

LED_SERVER = SocketConfig(socket_config["LED_SERVER"]["host"], socket_config["LED_SERVER"]["port"])
LOGIC_SERVER = SocketConfig(socket_config["LOGIC_SERVER"]["host"], socket_config["LOGIC_SERVER"]["port"])
IO_SERVER = SocketConfig(socket_config["IO_SERVER"]["host"], socket_config["IO_SERVER"]["port"])

File: android/app/src/main/java/com/cubi/app/resources/SocketConfig.java
package com.cubi.app.resources;

import org.json.JSONArray;
import org.json.JSONException;
import org.json.JSONObject;

import java.io.FileInputStream;
import java.io.FileNotFoundException;
import java.io.FileReader;
import java.io.IOException;
import java.io.InputStream;
import java.io.InputStreamReader;
import java.io.Reader;

public class SocketConfig {

    private final static String configLocation = "com/cubi/app/resources/server-config.json";
    private final static String json = readAllText(configLocation);

    public static Config getConfig(String serverName) {
        Config config = null;

        try {
            JSONObject obj = new JSONObject(json);
            JSONObject obj2 = obj.getJSONObject(serverName);

            config = new Config(obj2.getString("host"), obj2.getInt("port"));

        } catch (JSONException e) {
            e.printStackTrace();
        }

        return config;
    }

    private static String readAllText(String file)  {

        StringBuilder sb = new StringBuilder(512);
        FileInputStream is = null;
        try {
            is = new FileInputStream(file);
        } catch (FileNotFoundException e) {
            e.printStackTrace();
        }

        try {
            Reader r = new InputStreamReader(is, "UTF-8");
            int c = 0;
            while ((c = r.read()) != -1) {
                sb.append((char) c);
            }
        } catch (IOException e) {
            throw new RuntimeException(e);
        }
        return sb.toString();
    }
}
